library/NotificationCenter/AutoSuggester.php

```diff
@@ -42,7 +42,7 @@ public static function load($dc)
         static::$strType = static::$objNotification->type;
 
         foreach ($GLOBALS['TL_DCA'][static::$strTable]['fields'] as $field => $arrConfig) {
-            if ('nc_tokens' === $arrConfig['eval']['rgxp']) {
+            if (isset($arrConfig['eval']['rgxp']) && 'nc_tokens' === $arrConfig['eval']['rgxp']) {
```

Comment:
```suggestion
            if ('nc_tokens' === $arrConfig['eval']['rgxp'] ?? null) {
```

Reply:
Unfortunately, there is still support for `php ^5.4` in `composer.json` - this is not compatible - shall we update the `php` requirement?